.travis.yml

```diff
@@ -16,6 +17,9 @@ jobs:
         - npx lockfile-lint --type npm --path package-lock.json -o "https:" -o "file:" --allowed-hosts npm
       install:
         - npm ci
+      before-script:
+        - psql -c 'create database postmangovsg_test_1;' -U postgres
+        - psql -c 'create database postmangovsg_test_2;' -U postgres
```

Comment:
Would it make sense to do this initialisation and the subsequent teardown in the `globalSetup` and the `globalTeardown` step? Just thinking along of lines of making it easier when running in dev so that he/she wouldn't have to remember to create/drop the two test db.